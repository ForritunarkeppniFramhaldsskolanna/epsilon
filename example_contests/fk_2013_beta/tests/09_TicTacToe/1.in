4
3
.OX
OOX
X.X
3
O.O
XOX
XXO
5
O...X
O...X
O...X
O...X
O...X
4
...O
..OX
..XX
O..X
