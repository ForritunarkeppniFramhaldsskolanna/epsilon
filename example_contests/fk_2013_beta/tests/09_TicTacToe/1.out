X won
O won
Error
Neither won
